src/DetailPage.js

```diff
@@ -0,0 +1,116 @@
+import Header from "./Header"
+import DetailPageButton from "./DetailPageButton";
+import axios from 'axios';
+import React, { useState, useEffect } from 'react';
+
+export default function DetailPage(props) {
+    const {themeChanger, darkMode} = props;
+    const [country, setCountry] = useState([])
+    const [borderNames, setBorderNames] = useState([])
+
+    useEffect(()=>{
+        axios.get("https://restcountries.com/v3.1/alpha/BE")
```

Comment:
It's a good practice to store your base URL in an environment variable. Create a .env file in the root of your project if you don't already have one. In the .env file, define your base URL as an environment variable.You can access this environment variable using process.env